File: src/main/java/org/example/kotlinWorkshop/java/lesson03/t2_map/t2_2/Main.java
package org.example.kotlinWorkshop.java.lesson03.t2_map.t2_2;

import java.util.HashMap;
import java.util.Map;

public class Main {

  public static void main(String[] args) {
    var classWithMutableMap = new ClassWithMutableMap(
        new HashMap<>(Map.of("some_key", "some_value", "another_key", "another_value"))
    );

    classWithMutableMap.getMap().put("another_key3", "anotherValue");

    System.out.println(classWithMutableMap.getMap());
    System.out.println(classWithMutableMap.getMap().get("some_key"));
  }
}

File: src/main/kotlin/org/example/kotlinWorkshop/kotlin/lesson03/t2_map/t2_2/Main.kt
package org.example.kotlinWorkshop.kotlin.lesson03.t2_map.t2_2

fun main() {
    val classWithMutableMap = ClassWithMutableMap(
        mutableMapOf("some_key" to "some_value", "another_key" to "another_value")
    )

    classWithMutableMap.map.put("another_new_key", "new_value")
    classWithMutableMap.map["new_key"] = "new_value"

    println(classWithMutableMap.map)
}